src/main/java/com/microfocus/application/automation/tools/UftToolUtils.java

```diff
@@ -0,0 +1,152 @@
+/*
```

Comment:
Please move this method to uft package.
* From now on, all new classes that would be created by uft should be in uft package.